spark-bigquery-tests/src/main/java/com/google/cloud/spark/bigquery/integration/TestConstants.java

```diff
@@ -67,7 +68,23 @@ public class TestConstants {
   static final DecimalType NUMERIC_SPARK_TYPE =
       DataTypes.createDecimalType(BQ_NUMERIC_PRECISION, BQ_NUMERIC_SCALE);
   static final String SHAKESPEARE_TABLE = "bigquery-public-data.samples.shakespeare";
+  static final String BIGLAKE_SHAKESPEARE_TABLE = "spark_bq_connector_dataset.biglake-shakespeare";
   static final long SHAKESPEARE_TABLE_NUM_ROWS = 164656L;
+  static final String TEMPORARY_GCS_BUCKET_ENV_VARIABLE = "TEMPORARY_GCS_BUCKET";
+  static final String BIGLAKE_CONNECTION_ID_ENV_VARIABLE = "BIGLAKE_CONNECTION_ID";
+  static final String temporaryGcsBucket =
```

Comment:
const should adhere to the name standard: `TEMPORARY_GCS_BUCKET`

Reply:
Addressed